src/constants/network.ts

```diff
@@ -1 +1,11 @@
-export type NetworkId = 1 | 42;
+export type NetworkId = 1 | 3 | 4 | 5 | 42;
+
+export enum Network {
+  MAINNET = 1,
+  ROPSTEN = 3,
+  RINKEBY = 4,
+  GÖRLI = 5,
+  KOVAN = 42
```

Comment:
I actually really liked how this was done on Uniswap, so I adopted it too. (Having named network ids)